charmverse/app.charmverse.io
# Allow forums to be made public

To test,

Go to a forum home as a space admin, select a category, click "manage permissions" and toggle the "Public" option.

Create a post within this category.

Go to the space forum home from a private browsing window. You should see the public category, and the post
